web/src/pages/FilesPage.spec.tsx

```diff
@@ -26,6 +26,9 @@ describe('FilesPage', () => {
   const originalWindowOpen = window.open;
   afterEach(() => {
     window.open = originalWindowOpen;
+    window.HTMLElement.prototype.scrollIntoView = () => {
```

Comment:
I'm guessing this is supposed to be in the `beforeEach`, and it just happens to work because of the order the tests are run in, right?